sdk/storage/CMakeLists.txt

```diff
@@ -99,3 +99,5 @@ endif()
 
 # make sure that users can consume the project as a library.
 add_library (azure::storage ALIAS azure-storage)
+
```

Comment:
Ditto on hardcoded version

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/217